server/weaverbird/backends/sql_translator/steps/formula.py

```diff
@@ -0,0 +1,64 @@
+from distutils import log
+
+from weaverbird.backends.sql_translator.steps.utils.query_transformation import (
+    build_selection_query,
+    complete_fields,
+    handle_zero_division,
+)
+from weaverbird.backends.sql_translator.types import (
+    SQLPipelineTranslator,
+    SQLQuery,
+    SQLQueryDescriber,
+    SQLQueryRetriever,
+)
+from weaverbird.pipeline.steps import FormulaStep
+
+
+def translate_formula(
+    step: FormulaStep,
+    query: SQLQuery,
+    index: int,
+    sql_query_retriever: SQLQueryRetriever = None,
+    sql_query_describer: SQLQueryDescriber = None,
+    sql_translate_pipeline: SQLPipelineTranslator = None,
+) -> SQLQuery:
+    """
+    Supported operators are:
+    - "+"
+    - "-"
+    - "*"
+    - "/" -> To avoid
+    - "%"
```

Comment:
If operator "%" is supported, I lthink it is a good idea to write a test :)

Reply:
Done